src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/LookupTests.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Net.Http;
+using Altinn.Platform.Storage.Client;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Controllers;
+using Altinn.Platform.Storage.IntegrationTest.Fixtures;
+using Altinn.Platform.Storage.Models;
+using Altinn.Platform.Storage.Repository;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Storage.Interface.Clients;
+using Xunit;
+
+namespace Altinn.Platform.Storage.IntegrationTest
+{
+    /// <summary>
+    /// Tests to se if integration with Bridge works.
+    /// </summary>
+    public class LookupTests : IClassFixture<PlatformStorageFixture>
+    {
+        private readonly PlatformStorageFixture fixture;
+        private readonly HttpClient client;
+        private readonly InstanceClient storageClient;
+        private readonly string versionPrefix = "/storage/api/v1";
+
+        /// <summary>
+        /// Lookup tests
+        /// </summary>
+        /// <param name="fixture">the fixture to simulate system under test</param>
+        public LookupTests(PlatformStorageFixture fixture)
+        {
+            this.fixture = fixture;
+            this.client = this.fixture.Client;
+            this.storageClient = new InstanceClient(this.client);
+        }
+
+        /// <summary>
+        /// Check instance owner lookup.
+        /// </summary>
+        [Fact]
+        public async void InstanceOwnerLookup()
```

Comment:
If there come a time to include more tests, a non-happy path test should be included.